hangman/lib/hangman/move.ex

```diff
@@ -0,0 +1,55 @@
+defmodule Hangman.Move do
+
+  # try to specify that guess should a String with length of one
+  # but guard clauses forbid invoking my functions or even String.length
+  def make_move(game = %Hangman.Struct{}, guess) when is_bitstring(guess) do
+    #once a movement is made
+    #fisr update used and last_guess
+    last_guess = guess
+    new_used   =
+      case guess in game.used do
+	true   -> game.used
+	false  -> game.used ++ List.wrap(guess)
+      end
+    new_game   = %Hangman.Struct{ game | used: new_used, last_guess: last_guess}
+    final_game = is_in(guess in new_game.word, new_game, guess)
+    { final_game, Hangman.Tally.tally(final_game) }
+  end
+
+  def make_move(_,_) do
```

Comment:
What would happen if you didn't implement this function? 

The code would raise an exception, just as you do. The error wouldn't be as specific as yours, but the effect would be the same. That is more idiomatic.